### DISCUSSION TITLE
Oauth with no pin

### DISCUSSION THREAD

**Discussion by mmoore99** [henrikfroehling/Trakt.NET#598] [Help] [answered]
It appears that Trakt no longer supports oauth using a pin. How should one go about authenticating without the ability to get a pin?

**Comment by henrikfroehling** [comment, marked as answer]
Sorry for the late response.
I just tested the OAuth workflow and the Trakt PIN seems to be still working.
![Trakt tv-PIN-Code](https://github.com/henrikfroehling/Trakt.NET/assets/3494788/096b1521-e44c-42f3-aece-ceecd3d57dc0)
![Trakt tv-OAuth](https://github.com/henrikfroehling/Trakt.NET/assets/3494788/c5e3821b-988c-473d-a9e6-34861e727377)
